internal/common/metrics/constants.go

```diff
@@ -22,78 +22,78 @@ package metrics
+	ActivityPollTransientFailedCounter = TemporalMetricsPrefix + "activity-poll-transient-failed"
+	ActivityPollNoTaskCounter          = TemporalMetricsPrefix + "activity-poll-no-task"
+	ActivityPollSucceedCounter         = TemporalMetricsPrefix + "activity-poll-succeed"
+	ActivityPollLatency                = TemporalMetricsPrefix + "activity-poll-latency"
+	ActivityScheduledToStartLatency    = TemporalMetricsPrefix + "activity-scheduled-to-start-latency"
+	ActivityExecutionFailedCounter     = TemporalMetricsPrefix + "activity-execution-failed"
+	ActivityExecutionLatency           = TemporalMetricsPrefix + "activity-execution-latency"
+	ActivityResponseLatency            = TemporalMetricsPrefix + "activity-response-latency"
+	ActivityResponseFailedCounter      = TemporalMetricsPrefix + "activity-response-failed"
+	ActivityEndToEndLatency            = TemporalMetricsPrefix + "activity-endtoend-latency"
+	ActivityTaskPanicCounter           = TemporalMetricsPrefix + "activity-task-panic"
+	ActivityTaskCompletedCounter       = TemporalMetricsPrefix + "activity-task-completed"
+	ActivityTaskFailedCounter          = TemporalMetricsPrefix + "activity-task-failed"
+	ActivityTaskCanceledCounter        = TemporalMetricsPrefix + "activity-task-canceled"
+	ActivityTaskCompletedByIDCounter   = TemporalMetricsPrefix + "activity-task-completed-by-id"
+	ActivityTaskFailedByIDCounter      = TemporalMetricsPrefix + "activity-task-failed-by-id"
+	ActivityTaskCanceledByIDCounter    = TemporalMetricsPrefix + "activity-task-canceled-by-id"
+	LocalActivityTotalCounter          = TemporalMetricsPrefix + "local-activity-total"
+	LocalActivityTimeoutCounter        = TemporalMetricsPrefix + "local-activity-timeout"
+	LocalActivityCanceledCounter       = TemporalMetricsPrefix + "local-activity-canceled"
+	LocalActivityFailedCounter         = TemporalMetricsPrefix + "local-activity-failed"
+	LocalActivityPanicCounter          = TemporalMetricsPrefix + "local-activity-panic"
+	LocalActivityExecutionLatency      = TemporalMetricsPrefix + "local-activity-execution-latency"
+	WorkerPanicCounter                 = TemporalMetricsPrefix + "worker-panic"
 
-	UnhandledSignalsCounter = CadenceMetricsPrefix + "unhandled-signals"
-	CorruptedSignalsCounter = CadenceMetricsPrefix + "corrupted-signals"
+	UnhandledSignalsCounter = TemporalMetricsPrefix + "unhandled-signals"
+	CorruptedSignalsCounter = TemporalMetricsPrefix + "corrupted-signals"
 
-	WorkerStartCounter = CadenceMetricsPrefix + "worker-start"
-	PollerStartCounter = CadenceMetricsPrefix + "poller-start"
+	WorkerStartCounter = TemporalMetricsPrefix + "worker-start"
+	PollerStartCounter = TemporalMetricsPrefix + "poller-start"
 
-	CadenceRequest        = CadenceMetricsPrefix + "request"
-	CadenceError          = CadenceMetricsPrefix + "error"
-	CadenceLatency        = CadenceMetricsPrefix + "latency"
-	CadenceInvalidRequest = CadenceMetricsPrefix + "invalid-request"
+	TemporalRequest        = TemporalMetricsPrefix + "request"
```

Comment:
There are few places like this where we don't actually need `Temporal` or `Cadence` to be the part of the var/func/object name. Also there are a lot of "temporal" in comments where it can be safely removed (like "temporal server" => "server")

Reply:
Agreed.  I'll do that as a separate PR.